public/app/features/dashboard/components/TransformationsEditor/TransformationsEditor.tsx

```diff
@@ -363,16 +536,52 @@ class UnThemedTransformationsEditor extends React.PureComponent<TransformationsE
     }
 
     return (
-      <CustomScrollbar autoHeightMin="100%">
-        <Container padding="md">
+      <CustomScrollbar scrollTop={this.state.scrollTop} autoHeightMin="100%">
+        <Container padding="lg">
           <div aria-label={selectors.components.TransformTab.content}>
             {hasTransforms && alert ? (
               <Alert
                 severity={AppNotificationSeverity.Error}
                 title="Transformations can't be used on a panel with alerts"
               />
             ) : null}
-            {hasTransforms && this.renderTransformationEditors()}
+            {hasTransforms && config.featureToggles.transformationsRedesign && !this.state.showPicker && (
+              <div
+                className={css`
+                  display: flex;
+                  justify-content: space-between;
+                  margin-bottom: 24px;
+                `}
+              >
+                <span
+                  className={css`
+                    font-size: 16px;
```

Comment:
lots of inline styles laying around, maybe move css into classes and pull the styles out of the react/jsx?

Reply:
Removed all inline styles